x-pack/test/functional/apps/saved_objects_management/multi_space_import.ts

```diff
@@ -109,14 +73,43 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     it('imported dashboard into default space should render correctly', async () => {
+      await PageObjects.settings.navigateTo();
+
+      await PageObjects.settings.clickKibanaSavedObjects();
+
+      const initialObjectCount = await PageObjects.savedObjects.getExportCount();
+
       const spaceId = 'default';
-      await importIntoSpace(spaceId);
+      const importFileName = '_8.0.0_multispace_import.ndjson';
+      const importFilePath = path.join(__dirname, 'exports', importFileName);
+
+      const newObjectCount = await PageObjects.savedObjects.importIntoSpace(
+        importFilePath,
+        spaceId
```

Comment:
Just a nit, but since you made `importIntoSpace` default to the 'default' space, you really don't have to pass it in.  But maybe it makes the test more readable to leave it anyway since elsewhere in the test you import into non-default spaces.

Reply:
I think I like it being explicitly clear @LeeDr :) Thanks!